client/src/components/Sidebar/menu/SubMenuItem.tsx

```diff
@@ -64,10 +72,16 @@ export const SubMenuItem: React.FunctionComponent<SubMenuItemProps> = ({
     'sidebar-sub-menu-trigger-icon' +
     (isOpen ? ' sidebar-sub-menu-trigger-icon--open' : '');
 
+  const dismissibleCount = item.menuItems.reduce(
+    (acc, curr) => (isDismissed(curr, state) ? acc : acc + 1),
+    0,
+  );
```

Comment:
If we’re just doing a count I think this would be clearer as:

```suggestion
  const dismissibleCount = item.menuItems.filter(subItem => !isDismissed(subItem, state)).length;
```